mkosi

```diff
@@ -3633,10 +3632,7 @@ def build_stuff(args: CommandLineArguments) -> None:
     signature = calculate_signature(args, checksum)
     bmap = calculate_bmap(args, raw)
 
-    link_output(args,
-                workspace.name,
-                raw.name if raw is not None else None,
-                tar.name if tar is not None else None)
+    link_output(args, workspace, raw or tar)
```

Comment:
hmm, how come this .name is dropped here?

Reply:
`.name` is used in `link_output`.